knowledge_repo/app/routes/index.py

```diff
@@ -137,12 +136,14 @@ def render_cluster():
         author_to_posts = {}
         authors = (db_session.query(User).all())
         for author in authors:
-            author_posts = [post for
-                            post in author.posts
-                            if post.is_published and not post.contains_excluded_tag]
+            author_posts = [
+                [post.title, True, 0, post]
```

Comment:
For clarity, can you convert this to use a NamedTuple instance?

Perhaps something like: `ClusterEntry = NamedTuple('name', 'is_post', 'children_count', 'content')`